k8s/client.go

```diff
@@ -230,13 +231,21 @@ func (client *Client) FindServicesByLabel(ctx context.Context, label, ns string)
 }
 
 func (client *Client) UpdateDeployment(ctx context.Context, name, image, version string) error {
-	deployment, err := client.FindDeploymentByName(ctx, name)
+	deploymentList, err := client.FindDeploymentByLabel(ctx, name)
```

Comment:
Is it expected a name (single string or a label)? If its a label, make sure to construct it here. 			label := fmt.Sprintf("%s=%s", k8s.LabelAggregatorID, agg.ID)

Reply:
i changed the fucntion and parameter name